app/controllers/content_items_controller.rb

```diff
@@ -22,6 +23,40 @@ def show
     render_template
   end
 
+  def choose_sign_in
+    set_up_self_assessment_ab_test
+    content_item = Services.content_store.content_item(content_item_path)
+    @content_item = ContentItemPresenter.new(content_item, content_item_path)
+    @error = params[:error]
+    render template: 'content_items/signin/choose-sign-in'
+  end
+
+  def not_registered
+    set_up_self_assessment_ab_test
+    content_item = Services.content_store.content_item(content_item_path)
+    @content_item = ContentItemPresenter.new(content_item, content_item_path)
+    render template: 'content_items/signin/not-registered'
+  end
+
+  def lost_account_details
+    set_up_self_assessment_ab_test
+    content_item = Services.content_store.content_item(content_item_path)
+    @content_item = ContentItemPresenter.new(content_item, content_item_path)
+    render template: 'content_items/signin/lost-account-details'
+  end
+
+  def sign_in_options
+    if params["sign-in-option"] == "government-gateway"
```

Comment:
This could be a case/when statement

Reply:
We could have. Is there a benefit to using a case statement?

Reply:
DRY principle, but you don't need to address it as it will all be reverted in a week